### PR TITLE
HL-50

### PR DESCRIPTION
**Pull request by JasminFarhi**
[[Link to ticket](url)](https://dltx.atlassian.net/jira/software/projects/HL/boards/82?selectedIssue=HL-50)
Screen Shot 2022-11-04 at 10 49 22 am

### CODE REVIEW

File: src/components/Modal.tsx [hunk 1]
```diff
@@ -36,6 +36,14 @@ const Modal = ({ isOpen, onClose, children }: ModalProps) => {
                 leaveTo="opacity-0 scale-95"
               >
                 <Dialog.Panel className="transform overflow-hidden rounded-2xl bg-white p-6 text-left align-middle shadow-xl transition-all">
+                  <div className="w-full -mb-6 relative flex justify-end">
+                    <button
+                      className=" text-gray-300 cursor-pointer hover:text-gray-700 focus:outline-none"
```
Comment (brettvniekerk) [inline src/components/Modal.tsx:41]: please remove the blank space at the start of this string

Comment (brettvniekerk) [reply src/components/Modal.tsx:41]: also `cursor-pointer` here is redundant as the `<button>` tag already makes the cursor a pointer

File: src/components/Modal.tsx [hunk 2]
```diff
@@ -36,6 +36,14 @@ const Modal = ({ isOpen, onClose, children }: ModalProps) => {
...
+                      onClick={onClose}
+                    >
+                      CLOSE
```
Comment (brettvniekerk) [inline src/components/Modal.tsx:44]: the ticket specifies an `X`, not `CLOSE` - if the ticket needs changing please update it

File: src/pages/Bets/components/BetModal.tsx [hunk 3]
```diff
@@ -19,7 +19,6 @@ const BetModal = ({ isOpen, onClose, betData }: Props) => {
 };
 
 export default BetModal;
-
```
Comment (brettvniekerk) [inline src/pages/Bets/components/BetModal.tsx:22]: please leave the space here

**Review by brettvniekerk — APPROVED** [review]

**Review by xiangzhe0812 — APPROVED** [review]

**Review by pondpiu — APPROVED** [review]

**Pull request merged**